client/components/deposits-overview/deposit-notices.tsx

```diff
@@ -150,3 +150,37 @@ export const NegativeBalanceDepositsPausedNotice: React.FC = () => (
 		} ) }
 	</InlineNotice>
 );
+
+/**
+ * Renders a notice informing the user that deposits only occur when there are funds available.
+ */
+export const NoFundsAvailableForDepositNotice: React.FC = () => (
+	<InlineNotice
+		status="warning"
+		icon
+		className="no-funds-available-for-deposit-notice"
+		isDismissible={ false }
+	>
```

Comment:
Suggestion: remove className that deosn't look like it's being used.
```suggestion
	<InlineNotice status="warning" icon isDismissible={ false }>
```